plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtMigrateCommandWrapper.java

```diff
@@ -48,6 +49,9 @@
 @ResourceWrapper(handles =  MigrateCommand.class)
 public final class LibvirtMigrateCommandWrapper extends CommandWrapper<MigrateCommand, Answer, LibvirtComputingResource> {
 
+    private static final String CONTENTS_WILDCARD = ".*";
+    private static final String GRAPHICS_ELEM_END = "/graphics>";
```

Comment:
why would that be needed, the match is conclusive enough without it. Adding it would be purely cosmetic, wouldn't it?